agent/bench-scripts/pbench-user-benchmark

```diff
@@ -32,16 +32,20 @@ benchmark="pbench-user-benchmark"
 # Defaults
 config=""
 tool_group=default
+sysinfo=default
+check=true
 
 # Process options and arguments
-opts=$(getopt -q -o C: --longoptions "config:,tool-group:" -n "getopt.sh" -- "$@");
+opts=$(getopt -q -o C: --longoptions "config:,tool-group:,sysinfo:" -n "getopt.sh" -- "$@");
 if [ $? -ne 0 ]; then
 	printf -- "$*\n"
 	printf "\n"
 	printf "\t${benchmark}: you specified an invalid option\n\n"
 	printf "\tThe following options are available:\n\n"
 	printf -- "\t\t-C str --config=str            name of the test config\n"
 	printf -- "\t\t       --tool-group=str\n"
+	printf -- "\t\t       --sysinfo=str,          str = comma seperated values of sysinfo to be collected\n"
+	printf -- "\t                                       (available: default, all, none, sos, kernel_config, libvirt, topology, block)\n"
```

Comment:
Rather than repeat the list of available options here, perhaps we could add an option to pbench-collect-sysinfo which will emit them for the usage string here?